### PR TITLE
update k8s readme

### PR DESCRIPTION
**Pull request by siddharthist**
- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release
- [x] Updates documentation relevant to the changes

 * note that services can't be exposed to the internet
 * add DNS outline from Asteris's internal wiki

### CODE REVIEW

File: roles/kubernetes/README.rst [hunk 1]
```diff
@@ -23,3 +23,14 @@ To talk to the services launched inside Kubernetes, launch them with the
 <https://aster.is/blog/2016/03/11/the-hamburger-of-kubernetes-service-types/>`_),
 then connect on the assigned port on any Kubernetes worker node. Consul service
 integration will happen in a future release.
+
+Currently, services cannot be exposed to the internet (tracked on issue #1429).
```
Comment (sehqlr) [inline roles/kubernetes/README.rst]: Should this be a link to the issue?

Comment (ryane) [inline roles/kubernetes/README.rst]: this line can actually be removed now as #1429 is resolved.

Comment (siddharthist) [inline roles/kubernetes/README.rst]: :fire:

**Comment by avnik** [comment]
Need to be rebased before merge,  this README updated in master

**Comment by avnik** [comment]
@siddharthist I  rebase branch, and make it mergable. Take a quick look if you ok with order of sections, and I'll merge it

**Comment by siddharthist** [comment]
@avnik Just made one quick fix (removing the reference to an outdated ticket). Thanks for the review!

**Pull request merged**